plugins/inputs/conntrack/conntrack.go

```diff
@@ -50,7 +61,7 @@ func (c *Conntrack) setDefaults() {
 	}
 }
 
-func (*Conntrack) SampleConfig() string {
+func (c *Conntrack) SampleConfig() string {
```

Comment:
Please revert this change as you don't need to access the `Conntrack` instance nor any fields.